shell/platform/tizen/external_texture_pixel_gl_stub.cc

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
```

Comment:
```suggestion
// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
```
Now we should use `2021` for all new source code written this year. :)